syntax = "proto3";
package m2.pb;

message Vec2f {
  float x = 1;
  float y = 2;
}
